guides/releasing/Release Workflow.md

```diff
@@ -357,27 +449,82 @@
 - Commit and push changes to release branch.
 - Cherry pick this commit to `master` and push.
 - Open this commit on our CI. Go to the `docs` workflow and approve `docs_approve_deploy` job that starts `docs_deploy` job - keep an eye on it and make sure it gets deployed successfully.
+- The Algolia config needs to be updated in order for the new docs to be searchable. TODO: add instructions for that here. For now ping @cruzach or @byCedric about this.
 
-# Stage 7 - Snack
+## 6.4. Promote versions to production
 
-## 7.1. Add SDK support to Snack
+| Prerequisites                                             |
+| --------------------------------------------------------- |
+| [6.1. Release iOS/Android clients](#61-release-iosandroid-clients) |
+| [6.2. Deploy Turtle/ExpoKit to production](#62-deploy-turtleexpokit-to-production) |
+| [6.3. Generate and deploy new docs](#63-generate-and-deploy-new-docs) |
 
-**Why:** Once the new SDK is available publicly, we need to update Snack to support that SDK.
+**Why:** It's time for everything that uses the production versions endpoint to know about this new SDK version!
 
-**How:** For now the only responsible person for this task is @tc - coordinate this with him.
+**How:** 
+
+- `et promote-versions-to-prod`
+- Double check every change before pressing `y`!
 
-# Stage 8 - Press release
+## 6.5. Promote packages to `latest` on NPM registry
 
-## 8.1. Publish a blog post
+**Why:** Previously we've published packages and now, if everything works like a charm, we can promote those packages to `latest` on NPM.
+
+**How:**
+
+- TODO: need a script for this. For now talk to @tsapeta
```

Comment:
Yeah, I have it almost done, gonna submit a PR soon 😉 
It will be just `et promote-packages`